File: TestDDDExpress/infrastructure/personRepository.js
const {DataTypes, Sequelize} = require('sequelize');

class PersonRepository {
    sequelize = new Sequelize('TestDDD_DB', 'sa', 'Nada1234*', { host: 'localhost', dialect: 'mssql' });
    person = this.sequelize.define('persons', {
        identification: {
            type: DataTypes.STRING,
            primaryKey: true
        },
        firstNames: DataTypes.STRING,
        lastNames: DataTypes.STRING,
        age: DataTypes.STRING,
        gender: DataTypes.NUMBER,
    }, {
        timestamps: false
    });

    constructor() {
    }

    getPeople = async () => {
        return await this.person.findAll()
    }

    getPerson = async (identification) => {
        return await this.person.findOne({where: {identification}})
    }

    createPerson = async (person) => {
        return await this.person.create(person)
    }

    updatePerson = async (identification, person) => {
        const personEntity = await this.getPerson(identification);
        return await personEntity.update(person)
    }

    deletePerson = async (identification) => {
        return await this.person.destroy({where: {identification}});
    }
}

module.exports = { PersonRepository }

File: TestDDD/TestDDD.Infrastructure/PersonRepository.cs
using TestDDD.Domain.Entities;
using TestDDD.Domain.Repositories;

namespace TestDDD.Infrastructure;

public class PersonRepository : IPersonRepository
{
    private readonly TestDDDContext _testDddContext;

    public PersonRepository(TestDDDContext testDddContext)
    {
        _testDddContext = testDddContext;
    }

    public PersonEntity GetPerson(string identification)
    {
        return this._testDddContext.Persons.Find(identification);
    }

    public void AddPerson(PersonEntity person)
    {
        this._testDddContext.Persons.Add(person);
        this._testDddContext.SaveChanges();
    }

    public void UpdatePerson(PersonEntity person)
    {
        this._testDddContext.Persons.Update(person);
        this._testDddContext.SaveChanges();
    }

    public void DeletePerson(PersonEntity person)
    {
        this._testDddContext.Remove(person);
        this._testDddContext.SaveChanges();
    }
}